lib/pact/command/exec_command.ex

```diff
@@ -220,6 +236,14 @@ defmodule Kadena.Pact.ExecCommand do
 
   @spec sign_commands(signs :: list(), cmd :: json_string_payload(), keypairs()) ::
           valid_sign_commands()
+  defp sign_commands(signs, cmd, []) when signs == [] do
```

Comment:
@EdwinGuayacan @Odraxs you can remove the guard and use pattern matching instead